src/dev_config.yaml

```diff
@@ -1,116 +1,52 @@
-  output_auto_outliers: False
-  output_outlier_qa : False
-  output_estimation_qa: False
-  output_apportionment_qa: False
-  output_long_form: False
-  output_short_form: False
-  output_gb_sas: False
-  output_ni_sas: False
-  output_tau: False
-  output_intram_by_pg_gb: False
-  output_intram_by_pg_uk: False
-  output_intram_by_itl1: False
-  output_intram_uk_itl2: False
-  output_intram_by_civil_defence: False
-  output_intram_by_sic: False
-  output_fte_total_qa: False
-  output_status_filtered: False
-  output_frozen_group: False
-years:
-  current_year: 2022 # TODO: put this in the userconfig
-  previous_years_to_load: 1 # TODO: put this in the userconfig
 runlog_writer:
-    write_csv: True # Write the runlog to a CSV file
-    write_hdf5: False # Write the runlog to an HDF5 file
-    write_sql: False # Write the runlog to a SQL database
-    display: False # Display the runlog in the terminal
-    log_path: "/ons/rdbe_dev/"
-hdfs_paths:
-  root: "/ons/rdbe_dev/"
-  logs_foldername: "/ons/rdbe_dev/logs/run_logs"
-  snapshot_path: "/ons/rdbe_dev/berd_survey/anonymised/v1/snapshot-202012-002-fba5c4ba-fb8c-4a62-87bb-66c725eea5fd.json"
-  secondary_snapshot_path: "/ons/rdbe_dev/berd_survey/anonymised/v1/NEW_SNAPSHOT_TODO.json"
-  postcode_masterlist: "/ons/rdbe_dev/mappers/2023/ONSPD_NOV_2022_UK.csv"
-  postcode_path: "/ons/rdbe_dev/staging/staging_qa/postcode_validation"
-  ultfoc_mapper_path: "/ons/rdbe_dev/mappers/2023/BERD_2022_ultfoc_anon.csv"
-
-  staging_output_path: "/ons/rdbe_dev/staging/staging_qa/full_responses_qa"
-  output_path:  "/ons/rdbe_dev/outputs"
-  cellno_2022_path: "/ons/rdbe_dev/mappers/2023/berd_2022_cellno_coverage.csv"
-  itl_mapper_path: "/ons/rdbe_dev/mappers/2023/itl.csv"
-  history_path: "/ons/rdbe_dev/BERD_V7_Anonymised"
-  backdata_path: ""
-  manual_outliers_path: "/ons/rdbe_dev/outliers/manual_outliers/manual_outlier_2023_09_09_sample_amonymised.csv"
-  imputation_path: "/ons/rdbe_dev/imputation"
-  manual_imp_trim_path: "/ons/rdbe_dev/imputation/manual_trimming"
-  outliers_path: "/ons/rdbe_dev/outliers"
-  estimation_path:  "/ons/rdbe_dev/estimation"
-  short_form_schema: "src/outputs/output_schemas/short_form_schema.toml"
-  long_form_schema: "src/outputs/output_schemas/long_form_schema.toml"
-  export_path: /ons/rdbe_dev/outgoing_export
-  feather_path: "/ons/rdbe_dev/staging/feather"
+  write_csv: True # Write the runlog to a CSV file
+  write_hdf5: False # Write the runlog to an HDF5 file
+  write_sql: False # Write the runlog to a SQL database
+  display: False # Display the runlog in the terminal
+  log_path: "/ons/rdbe_dev/"
 network_paths:
-  root: "R:/BERD Results System Development 2023/DAP_emulation/"
-  logs_foldername: "logs/run_logs"
-  snapshot_path: "R:/BERD Results System Development 2023/DAP_emulation/survey_return_data/snapshot-202212-002-83b5bacd-7c99-45cf-b989-d43d762dd054.json"
+  root: "R:/BERD Results System Development 2023/DAP_emulation/" # dev config
```

Comment:
Please could you put this back to how it was? I probably mislead you here with with the settings I asked you to copy from my devtest branch.

snapshot_path: "R:/BERD Results System Development 2023/DAP_emulation/survey_return_data/snapshot-202212-002-83b5bacd-7c99-45cf-b989-d43d762dd054.json"